stellarphot/gui_tools/seeing_profile_functions.py

```diff
@@ -384,12 +458,26 @@ def show_event(
                 x = int(np.floor(event.data_x))
                 y = int(np.floor(event.data_y))
 
-                rad_prof = CenterAndProfile(
-                    data,
-                    (x, y),
-                    profile_radius=profile_size,
-                    centering_cutout_size=centering_cutout_size,
-                )
+                try:
+                    rad_prof = CenterAndProfile(
+                        data,
+                        (x, y),
+                        profile_radius=profile_size,
+                        centering_cutout_size=centering_cutout_size,
+                    )
+                except RuntimeError as e:
```

Comment:
this exception is being flagged as untested...  I'll also note that this may be where you might have to tweak the code to avoid displaying multiple copies of the error message when a person repeated clicks far from a star [which caused the image to shift down as the error messages copies pile up].

Reply:
Also tested now...